packages/loader/container-loader/src/container.ts

```diff
@@ -1229,7 +1230,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             startConnectionP = this.connectToDeltaStream(connectionArgs);
             startConnectionP.catch((error) => { });
         }
-
+        const connection = await startConnectionP;
+        this.timestamp = connection?.maxMessageSize ?? 1;
```

Comment:
here I found connection will be IConnectionDetails, so I was thinking I can add timestamp to that interface as well?
I used .maxMessageSize just for an example

Reply:
I'm not sure what exactly you are trying to accomplish here. Are you looking for time stamp generated by ordering service (PUSH)? Why?

I think for the purpose of tracking all we care about is time (generated by client via performance.now(), maybe formatted to be human readable) to be generated and recorded in file metadata, preserved through summaries and logged into telemetry on file open. That's pretty much it.